info/v2/container.go

```diff
@@ -262,13 +262,23 @@ type TcpStat struct {
 	Closing     uint64
 }
 
+type UdpStat struct {
```

Comment:
I realize we duplicate structs for TcpStat, but I would prefer if you removed this struct, and used v1.UdpStat in this file